static/templates/right_sidebar.hbs

```diff
@@ -31,7 +31,7 @@
             </a>
             <div class="only-visible-for-spectators">
                 <div class="realm-description">
-                    <p>{{ realm_description }}</p>
+                    <p>{{{ realm_description }}}</p>
```

Comment:
A few issues with this:
* We should apply the `rendered_markdown` CSS class to the `realm-description` div, and ideally have that not contain the signup/login buttons.
* I don't think we want that `<p>` tag.
* We need to run `static/js/rendered_markdown.js` on any in-app content before rendering it to the DOM to protect against XSS; see how that module is used in the existing places in the UI.